### PR TITLE
Fix Symfony deprecated feature usages

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

File: Form/Type/ChoiceFieldMaskType.php [hunk 1]
```diff
@@ -42,8 +43,14 @@ public function buildView(FormView $view, FormInterface $form, array $options)
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
-        parent::setDefaultOptions($resolver);
+        $this->configureOptions($resolver);
```
Comment (Soullivaneuh) [inline Form/Type/ChoiceFieldMaskType.php]: Should have a comment with which version of Symfony is supported here.

**Comment by lsmith77** [comment]
for 2.0 imho we should bump the minimum requirement to PHP 5.4

**Comment by dbu** [comment]
can we just drop php 5.3 support now? it fails and imo its ok to give up a dead version of php in a minor version. if there are critical bugfixes, they can be backported to the last branch running on 5.3.

**Comment by lsmith77** [comment]
like I said .. +1 from me

**Comment by Soullivaneuh** [comment]
> can we just drop php 5.3 support now?

PHP version support dropping should be on new major.

**Comment by lsmith77** [comment]
not sure I agree that it requires a new major but in this case we do have a new major version:
https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/master/composer.json#L44

**Comment by dbu** [comment]
php 5.4 does [not even have security support anymore](https://secure.php.net/supported-versions.php) - lets bump the version to 5.5 (like symfony 3 does).

**Comment by dbu** [comment]
wouter, do you want to try dropping php 5.4 support?

**Comment by patkar** [comment]
backref: sulu-io/sulu#2106

**Comment by dbu** [comment]
@WouterJ would be great if you could rebase this and check the cs issue travis-ci is showing. i am ok to drop php 5.3 support if that keeps failing on travis.

File: Form/Type/ChoiceFieldMaskType.php (hunk 1 above)
Comment (dbu) [inline Form/Type/ChoiceFieldMaskType.php]: fixed in #363

**Comment by dbu** [comment]
continued in #363

**Pull request closed without merging**